CHANGELOG.md

```diff
@@ -7,22 +7,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [3.1.2] - 2019-03-29 (Enhancements / BugFix Release)
```

Comment:
Please leave [Unreleased] in place, above this so we always point to a diff of any current release and changes in between

Reply:
Resolved